### PR TITLE
Dropbox feat

### PR DESCRIPTION
**Pull request by sohamsshah**
Accidentally the fork resolving #486 was deleted. This PR add Dropbox feature as described in #486 and it is exact clone of that.

### CODE REVIEW

File: apps/table.html [hunk 1]
```diff
@@ -255,6 +255,7 @@ <h3>Steps for uploading.</h3>
                                                 </div> </div>
                                                 <a title="Upload from google drive" href="#" id='gdriveUpload' onclick="googlePickerStart()"><i style="font-size: 1.2em;" class="fab fa-google-drive"></i></a>
                                                 <a href="#" id='urlswitch' onclick="$('#urlInput').val(''); switchToUrl();" > <span style="position: absolute; right:2em;" > <i class="fas fa-link"></i> Upload from URL</span></a>
+												<a title="Upload from Dropbox" href="#" id='dropboxUpload' onclick="dropboxChooserStart()"><i style="font-size: 1.2em;" class="fab fa-dropbox"></i></a>
```
Comment (birm) [inline apps/table.html:258]: I think you forgot to re-add the config and Dropbox api to this file. In the other PR you also added:

```
<script type="text/javascript" src="https://www.dropbox.com/static/api/2/dropins.js" id="dropboxjs"></script>
<script src="./loader/config.js"></script>
```

Comment (sohamsshah) [reply apps/table.html:258]: Whoopsie. My bad. Added in next commit :)

**Comment by birm** [comment]
I'm getting `TypeError: file.slice is not a function` on attempts, and the creation of empty files. Does this match your observations?

**Comment by sohamsshah** [comment]
> I'm getting `TypeError: file.slice is not a function` on attempts, and the creation of empty files. Does this match your observations?

I have tested by uploading files via dropbox. Though haven't got this error; need more clarity on how to test this issue. I suppose it must not happen due to dropbox, still open to any suggestions and improvements to the code.

**Comment by sohamsshah** [comment]
Hi Birm, I could not replicate this error. I am unsure about how to solve this :( Can you assist me a bit (if possible)?

**Pull request closed without merging**